supported_versions.md

```diff
@@ -0,0 +1,85 @@
+| Dependency | Minimum Version | Maximum Version | Auto-Instrumented |
+|------------|-----------------|-----------------|-----------------|
+| cloud.google.com/go/pubsub | v1.33.0 | v1.45.3 | true
+| github.com/99designs/gqlgen | v0.17.36 | v0.17.61 | true
+| github.com/IBM/sarama | v1.40.0 | v1.43.3 | true
+| github.com/Shopify/sarama | v1.38.1 | v1.43.3 | true
+| github.com/aws/aws-sdk-go | v1.44.327 | v1.55.5 | false
+| github.com/aws/aws-sdk-go-v2 | v1.20.3 | v2.0.0-preview.4+incompatible | true
+| github.com/aws/aws-sdk-go-v2/config | v1.18.21 | v1.28.7 | false
+| github.com/aws/aws-sdk-go-v2/service/dynamodb | v1.21.4 | v1.38.1 | false
+| github.com/aws/aws-sdk-go-v2/service/ec2 | v1.93.2 | v1.198.1 | false
+| github.com/aws/aws-sdk-go-v2/service/eventbridge | v1.20.4 | v1.36.1 | false
+| github.com/aws/aws-sdk-go-v2/service/kinesis | v1.18.4 | v1.32.8 | false
+| github.com/aws/aws-sdk-go-v2/service/s3 | v1.32.0 | v1.71.1 | false
+| github.com/aws/aws-sdk-go-v2/service/sfn | v1.19.4 | v1.34.2 | false
+| github.com/aws/aws-sdk-go-v2/service/sns | v1.21.4 | v1.33.8 | false
+| github.com/aws/aws-sdk-go-v2/service/sqs | v1.24.4 | v1.37.3 | false
```

Comment:
```suggestion
```

IMHO all these shouldn't show up here as they can be confusing for users reading it, as they don't map to a specific integration (the integration is just `github.com/aws/aws-sdk-go-v2`).